src/directory/file_watcher.rs

```diff
@@ -88,6 +88,9 @@ impl FileWatcher {
 
         Ok(hasher.finalize())
     }
+    pub fn unsubscribe(&self) {
```

Comment:
this should be terminate, as there is no resubcribe after that

Reply:
Make sense